examples/kubernetes/connectivity-check/resources.cue

```diff
@@ -0,0 +1,244 @@
+		if len(_command) > 0 {
+			command: _command
+		}
+		if _serverPort != "" {
+			env: [{
+				name:  "PORT"
+				value: _serverPort
+			}]
+		}
+		ports: [...{
+			_expose: *false | true
+		}]
+	}
+
+	_allowProbe:   [ "curl", "-sS", "--fail", "--connect-timeout", "5", "-o", "/dev/null", "\(_probeTarget)\(_probePath)"]
+	_rejectProbe:   [ "ash", "-c", "! curl -s --fail --connect-timeout 5 -o /dev/null \(_probeTarget)\(_probePath)"]
+	_c1: _container & {
+		name:           "\(_name)-container"
+		if _probeExpectFail {
+			readinessProbe: exec: command: _rejectProbe
+		}
+		if !_probeExpectFail {
+			readinessProbe: exec: command: _allowProbe
+		}
+	}
+	_containers: [_c1]
+
+	apiVersion: "apps/v1"
+	kind:       "Deployment"
+	metadata: {
+		name: _name
+		labels: {
+			name:      _name
+			topology:  *"any" | string
+			component: *"invalid" | string
+		}
+	}
+	spec: {
+		selector: matchLabels: name: _name
+		template: {
+			metadata: labels: name: _name
+			spec: containers: _containers
+			if _affinity != "" {
+				spec: affinity: podAffinity: requiredDuringSchedulingIgnoredDuringExecution: [{
+					labelSelector: matchExpressions: [{
+						key:      "name"
+						operator: "In"
+						values: [
+							_affinity,
+						]
+					}]
+					topologyKey: "kubernetes.io/hostname"
+				}]
+			}
+			if _antiAffinity != "" {
+				spec: affinity: podAntiAffinity: requiredDuringSchedulingIgnoredDuringExecution: [{
+					labelSelector: matchExpressions: [{
+						key:      "name"
+						operator: "In"
+						values: [
+							_antiAffinity,
+						]
+					}]
+					topologyKey: "kubernetes.io/hostname"
+				}]
+			}
+		}
+	}
+}
+
+deployment: [ID=_]: _spec & {
+	_name:         ID
+	_image:        string
+
+	// Expose services
+	_exposeClusterIP: *false | true
+	_exposeNodePort:  *false | true
+	_exposeHeadless:  *false | true
+
+	// Pod ports
+	_serverPort: *"" | string
+	if _serverPort != "" {
+		_probeTarget: "localhost:\(_serverPort)"
+	}
+
+	spec: {
+		replicas: *1 | int
+		template: spec: {
+			hostNetwork: *false | true
+		}
+	}
+}
+
+service: [ID=_]: {
+	_name:     ID
+	_selector: ID | string
+
+	apiVersion: "v1"
+	kind:       "Service"
+	metadata: {
+		name: ID
+		labels: {
+			name:      _name
+			topology:  *"any" | string
+			component: *"invalid" | string
+		}
+	}
+	spec: {
+		type: *"ClusterIP" | string
+		selector: name: _selector
+	}
+}
+
+_cnp: {
+	_name: string
+
+	apiVersion: "cilium.io/v2"
+	kind:       "CiliumNetworkPolicy"
+	metadata: {
+		name: _name
+		labels: {
+			name:      _name
+			topology:  *"any" | string
+			component: *"invalid" | string
+		}
+	}
+	spec: endpointSelector: matchLabels: name: _name
+}
+
+egressCNP: [ID=_]: _cnp & {
+	_name:      ID
+	_rules: [...{}]
+	_allowDNS: *true | false
+
+	// Implicitly open DNS visibility if FQDN rule is specified.
+	_enableDNSVisibility: *false | true
+	for r in _rules if len(r.toFQDNs) > 0 {
+		_enableDNSVisibility: true
+	}
+
+	if !_allowDNS {
+	spec: egress: _rules
```

Comment:
nit: missing indentation?

Reply:
I think I may have only run `cue fmt` on a later commit, can fix this up.